components/brave_rewards/browser/rewards_p3a.cc

```diff
@@ -14,6 +14,19 @@
 namespace brave_rewards {
 namespace p3a {
 
+#if !BUILDFLAG(IS_ANDROID)
+namespace {
+
+constexpr base::TimeDelta kMaxEnabledCauseTriggerTime = base::Minutes(1);
```

Comment:
Could probably use a short comment here explaining the purpose of this constant.